Automattic/wp-calypso
# Domains: Add Notice if User Already Owns a Custom Domain

#### Changes proposed in this Pull Request

* Adds a notice if the user already has a custom domain and is trying to register a new one. There's some more detailed context in the original issue, but basically a significant number of people actually intend to create a new site, but end up registering a domain for their current site

#### Testing instructions

Visit `/domains/add/` on a site with a custom domain, and verify that the notice appears. It should not appear if you don't have a custom domain.

<img width="1297" alt="Screenshot 2019-07-31 at 08 31 57" src="https://user-images.githubusercontent.com/43215253/62192692-f8e3da00-b36d-11e9-831a-e27021801d54.png">

@michelleweber, is this wording okay?

> You're purchasing an additional domain for (site title). Would you like to create a new site instead?
> Create a new site

Fixes #27932 and fixes #39829

cc @sgomes, @jsnajdr, @aidvu
